da/local/local.go

```diff
@@ -181,3 +181,8 @@ func (m *DataAvailabilityLayerClient) updateDAHeight() {
 func (d *DataAvailabilityLayerClient) GetMaxBlobSizeBytes() uint32 {
 	return maxBlobSize
 }
+
+func (m *DataAvailabilityLayerClient) GetSignerBalance() (*da.Balance, error) {
+	//TODO implement me
+	panic("implement me")
```

Comment:
TODO reminder

Reply:
same here, i think if this is not implemented it will panic when using mock da